packages/api/tests/lib/writeRecords/test-write-granules.js

```diff
@@ -4796,6 +4796,7 @@ test.serial('updateGranuleStatusToQueued() updates granule status in PostgreSQL/
     MaxNumberOfMessages: 2,
     WaitTimeSeconds: 10,
   });
+  console.log('Received Messages::', Messages);
```

Comment:
This test appears to be intermittently failing on main. Adding logging to help debug.

Reply:
Any update to this comment (over a month old)?

Reply:
Not really. This was from the intermittent unit test failure investigation, before discovering the logging improvements. I saw this test fail a few times but it's been a while. Removing the log as it's not actively being investigated.
https://github.com/nasa/cumulus/pull/3606/commits/8727d897552c6a455d7d2c9301e28621da82c0c9